### PR TITLE
Closes #998: compile ui tests on pre push

### LINKED ISSUE #998: Compile UI/integration tests during CI, local pre-push hooks
We don't run UI tests because they take forever but we should at least compile them to ensure they haven't broken.

### PR DESCRIPTION
**Pull request by Baron-Severin**
- Comments out compilation errors in ui tests
- Adds assembly (but not running) of ui tests to pre push hook

### CODE REVIEW

**Comment by Baron-Severin** [comment]
@AaronMT Is this PR alright with you?  It updates our pre push git hook to verify that all UI tests compile.  I commented out any current compilation errors, but otherwise made no changes to the tests.

If this will get in the way of your current work, we can hold off on this merge until it's done.

File: quality/pre-push-recommended.sh [hunk 1]
```diff
@@ -20,6 +20,8 @@
         detektCheck \
         testdebug
 
+./gradlew assembleAndroidTest
```
[the remark this answers is not in the document]
Comment (Baron-Severin) [reply quality/pre-push-recommended.sh]: Added and tested to make sure that it works as expected.

**Comment by mcomella** [comment]
@Baron-Severin Did we also set up automation to build the tests? We need to do that too before we close the issue.

**Comment by AaronMT** [comment]
Please hold off, I have fixes locally in place and other cleanup.

**Comment by Baron-Severin** [comment]
It looks like @AaronMT's fixes landed in https://github.com/mozilla-mobile/firefox-tv/pull/1163

File: .taskcluster.yml [hunk 2]
```diff
@@ -37,7 +37,8 @@ tasks:
           && git config advice.detachedHead false
           && git checkout {{event.head.sha}}
           && echo "--" > .adjust_token
-          && ./gradlew clean assemble lint checkstyle ktlint pmd detektCheck test
+          && ./gradlew clean assemble lint checkstyle ktlint pmd detektCheck test assembleAndroidTest
```
[the remark this answers is not in the document]
Comment (Baron-Severin) [reply .taskcluster.yml]: Good call

**Pull request merged**